src/components/Settings/SettingsNotifications.tsx

```diff
@@ -90,6 +91,17 @@ const SettingsNotifications: React.FC = ({ children }) => {
       route: '/settings/notifications/telegram',
       regex: /^\/settings\/notifications\/telegram/,
     },
+    {
+      text: intl.formatMessage(messages.webpush),
+      content: (
+        <span className="flex items-center">
+          <CloudUploadIcon className="h-4 mr-2" />
```

Comment:
```suggestion
          <GlobeAltIcon className="h-4 mr-2" />
```
I personally think the Cloud Upload icon looks a little odd / unbalanced, and we aren't uploading anything... what do you think of either the Globe Alt icon here?
![image](https://user-images.githubusercontent.com/52870424/115919744-4019d480-a447-11eb-956b-e08af542e40c.png)

Reply:
We are pushing notifications to the cloud. I think it fits better than just a globe?

Reply:
Maybe just the cloud icon, in that case? Since the notifications are actually sort of being pushed _from_ the cloud to devices, no?

Reply:
We push it to the cloud before it's distributed though, right?

Reply:
I suppose, lol. But does the user understand/care about that? The plain cloud would be less confusing and look better IMO 😄
![image](https://user-images.githubusercontent.com/52870424/115968215-bb8d8b80-a504-11eb-8d5f-e02d2399a7d0.png)

Reply:
I suppose, lol. But does the user understand/care about that? The plain cloud would be less confusing and look better IMO 😄
![image](https://user-images.githubusercontent.com/52870424/115968215-bb8d8b80-a504-11eb-8d5f-e02d2399a7d0.png)